template/lib.typ

```diff
@@ -86,7 +97,19 @@
   align(end, x.body)
   v(13pt)
   line(length: 100%, stroke: (paint: gray, thickness: 3.5pt))
-  v(15pt)
+  v(50pt)
+}
+
+#let main-content_heading(x) = {
+  pagebreak(weak: true)
+
+  set text(font: "TeX Gyre Heros", size: 20pt, stretch: 85%)
+  set block(spacing: 0pt)
+  v(75pt)
+  align(left, x)
+  v(13pt)
+  line(length: 100%, stroke: (paint: gray, thickness: 3.5pt))
+  v(50pt)
 }
```

Comment:
Why did you create another one? We already have main_heading. Also, isn't this PR about improving outlines?

Reply:
Well yes but I changed somethings related to the titles too

Reply:
I changed it in the new PR. Now that I understand the structure of data that typst uses ( .numbering, .body, etc) I see how stupid it was to use two separate functions hahahaha.